lib/hyrax/transactions/container.rb

```diff
@@ -109,6 +120,16 @@ class Container # rubocop:disable Metrics/ClassLength
         end
       end
 
+      namespace 'collection_resource' do |ops| # valkyrie collection
+        ops.register 'apply_collection_type_permissions' do
+          Steps::ApplyCollectionTypePermissions.new
+        end
+
+        ops.register 'save_acl' do
+          Steps::SaveAccessControl.new
```

Comment:
is this a real step (i couldn't find it here, or in the existing codebase)? if so, is it needed or could this be handled by the pub/sub setup?

Reply:
This uses the same [step](https://github.com/samvera/hyrax/blob/main/lib/hyrax/transactions/steps/save_access_control.rb) as used by [WorkCreate](https://github.com/samvera/hyrax/blob/main/lib/hyrax/transactions/work_create.rb#L16) ([registered in container](https://github.com/samvera/hyrax/blob/main/lib/hyrax/transactions/container.rb#L129-L131)).  This code registers the SaveAccessControl step for use with collections.

The CollectionCreate process follows the pattern established by WorkCreate.

Reply:
ah thanks, this is just me being confused.